app/controllers/users_controller.rb

```diff
@@ -54,7 +54,7 @@ def create
   def update
     respond_to do |format|
       if @user.update(user_params)
-        format.html { redirect_to “/boards/new”, notice: 'User was successfully updated.' }
+        format.html { redirect_to '/boards/new', notice: 'User was successfully updated.' }
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.